common/plugins/eu.esdihumboldt.hale.common.align/help/merge/help.xhtml

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" />
+<title>Merge</title>
+</head>
+<body>
+<div id ="help">
+	<h4>Example mapping:</h4>
+	<p>
+		The merge function is used to merge multiple source features within a single, source dataset. The merged source features create a single instance in the target dataset. In the merge configuration you must specify which instances should be merged together by specifying a merge key property. Instances that have the same value for this key property will be merged into one instance. For the other properties, the values are accumulated in the merged instance.<br>
```

Comment:
I think it would be good to mention that:
- multiple key properties can be selected (and that in that case all instances that are combined have the same values for all)
- it is also possible to not select a key property, which means all instances of that type get merged into one (which should be avoided if there is a large number of source instances)